intermediate/xarray_ecosystem.ipynb

```diff
@@ -10,26 +10,126 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    import cf_xarray.units  # must be imported before pint_xarray
</pre><p>the import order is not important anymore with recent versions of <code>pint</code>, so you should probably remove the comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>